### PR TITLE
Add log messages for locking, unlocking nodes and changing node tokens

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**
…okens

### CODE REVIEW

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java [hunk 1]
```diff
@@ -3242,6 +3244,9 @@ public void addNodeToken(String nodeUrl, String token) throws RMException {
             registerAndEmitNodeEvent(rmNode.createNodeEvent(RMEventType.NODE_STATE_CHANGED,
                                                             rmNode.getState(),
                                                             rmNode.getProvider().getName()));
+
+            logger.info("The token '" + token + "' is added to the node " + nodeUrl);
```
Comment (fviale) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java:3248]: you could add the caller as well in the message

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java [hunk 2]
```diff
@@ -3242,6 +3244,9 @@ public void addNodeToken(String nodeUrl, String token) throws RMException {
             registerAndEmitNodeEvent(rmNode.createNodeEvent(RMEventType.NODE_STATE_CHANGED,
                                                             rmNode.getState(),
                                                             rmNode.getProvider().getName()));
+
+            logger.info("The token '" + token + "' is added to the node " + nodeUrl + " by " + caller);
```
Comment (mboussaa) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java:3248]: caller. getName()?

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java [hunk 3]
```diff
@@ -3354,6 +3361,10 @@ public void setNodeTokens(String nodeUrl, List<String> tokens) throws RMExceptio
             registerAndEmitNodeEvent(rmNode.createNodeEvent(RMEventType.NODE_STATE_CHANGED,
                                                             rmNode.getState(),
                                                             rmNode.getProvider().getName()));
+
+            logger.info("The list of tokens of the node " + nodeUrl + "' are updated. The current list of tokens is: " +
```
Comment (mboussaa) [inline rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java:3365]: on the node is better?

Comment (mboussaa) [reply rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java:3365]: there is an extra single quote

Comment (mboussaa) [reply rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java:3365]: are updated => is updated

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java (hunk 1 above)
Comment (nebilbenmabrouk) [reply rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java]: Done.

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java (hunk 2 above)
Comment (nebilbenmabrouk) [reply rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java]: Done.

File: rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java (hunk 3 above)
Comment (nebilbenmabrouk) [reply rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java]: Done except for 'on the node is better?'. the current sentence is simpler for me

Comment (mboussaa) [reply rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java]: ok thx

**Review by mboussaa — APPROVED** [review]

**Review by fviale — APPROVED** [review]

**Pull request merged**